flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -73,9 +73,9 @@ public final class CatalogManager implements CatalogRegistry {
     private final Map<ObjectIdentifier, CatalogBaseTable> temporaryTables;
 
     // The name of the current catalog and database
-    private String currentCatalogName;
+    @Nullable private String currentCatalogName;
```

Comment:
very very nit: But could we agree on using this order?
```
private @Nullable String currentCatalogName;
```

The nullability should be next to the data type everywhere, not next to the visibility

Reply:
to me it looks weird, but I don't mind...